packages/fxa-auth-server/lib/payments/paypal-client.ts

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import querystring from 'querystring';
+import superagent from 'superagent';
+import pRetry from 'p-retry';
+
+export const PAYPAL_SANDBOX_API = 'https://api-3t.sandbox.paypal.com/nvp';
+export const PAYPAL_LIVE_API = 'https://api-3t.paypal.com/nvp';
```

Comment:
nice to have: these values as configs.  But also, I have no idea if they ever change.

Reply:
The API is deprecated, and hasn't changed in a decade or so.